### ISSUE TITLE
Split long running tests into separate repository

### ISSUE THREAD

**Issue by doerfli** [etherisc/gif-next#616]
Build times are too long (measured on a M2 Pro based Macbook Pro). 

Building all contracts inside the `contracts/` directory takes around 17 seconds. Building all contracts (including tests) takes around 4minutes. Removing all long running tests (Fuzz, Continuous) brings this down to 1m50s. 
Running all tests excluding long running tests takes about 2 seconds, running all of them (including long running) takes about 7 minutes. 

To reduce run times, we've already split running the long running tests on github into a separate task and run it only once a day. To further improve compilation times locally, it would make sense to split the long running tests into their own repository and ensure this is run once a day with the latest gif libraries to guarantee that the tests are still fully functional. Moving the long running tests into their own foundry project should improve build times for the main repo drastically.

**Comment by doerfli** [comment]
tests deleted. a copy is kept in branch https://github.com/etherisc/gif-next/tree/feature/long-running-tests
